app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -2096,10 +2104,14 @@ class AppointmentRepositoryAndroidTest {
       bpRepository.save(listOf(bloodPressureMeasurement)).blockingAwait()
 
       medicalHistoryAnswers?.run {
-        medicalHistoryRepository.save(patientUuid, OngoingMedicalHistoryEntry(
-            hasDiabetes = hasDiabetes,
-            diagnosedWithHypertension = hasHypertension
-        )).blockingAwait()
+        medicalHistoryRepository.save(
+            uuid = UUID.randomUUID(),
```

Comment:
Can we use an explicit `UUID` here, please?

Reply:
Done.